Gopkg.toml

```diff
@@ -1,7 +1,7 @@
 
 [[constraint]]
-  name = "github.com/bitrise-io/go-utils"
   branch = "master"
+  name = "github.com/bitrise-io/go-utils"
 
 [[constraint]]
   branch = "feature/app-icon"
```

Comment:
Do not forget to change it back to master after the merge is done for the package.